specifications/08-variants.json

```
@@ -394,6 +419,17 @@
                 "name": "disabled",
                 "enabled": false
             }
+        },
+        {
+            "description": "Feature.Variants.G should be disabled when gradual rollout is 0%, but has variants",
```

I think this better expresses what we're trying to test here, WDYT?

```suggestion
            "description": "Feature.Variants.G should be disabled when feature is calculated as false, but has variants",
```